website/frontend/src/components/HighlightsSection/index.js

```diff
@@ -7,79 +7,79 @@ import { loadHighlightsData, loadTagsData } from '../../../reduxStore/Highlights
 import ImageLoader from '../LoadSpinner/ImageLoader';
 
 const HighlightsSection = () => {
-    const dispatch = useDispatch();
-    const highlightsData = useHighlightsData();
+  const dispatch = useDispatch();
+  const highlightsData = useHighlightsData();
 
-    useEffect(() => {
-        dispatch(loadTagsData());
-        dispatch(loadHighlightsData());
-    }, [highlightsData.length]);
+  useEffect(() => {
+    dispatch(loadTagsData());
+    dispatch(loadHighlightsData());
+  }, [highlightsData.length]);
 
-    const highlights = highlightsData.slice(0, 3);
+  const highlights = highlightsData.slice(0, 3);
```

Comment:
Thanks @FaithDaka  for this.
I have a quick one.....this highlights data where we get a portion of it.....

1. Does it have an expiry date where things are automatically removed?
2. What happened to the previous highlights that existed? Who/what deletes them?